network/phonebook.go

```diff
@@ -104,48 +132,62 @@ func (e *phonebookEntries) filterRetryTime(t time.Time) []string {
 // new entries in they are being added
 // existing items that aren't included in they are being removed
 // matching entries don't change
-func (e *phonebookEntries) ReplacePeerList(they []string) {
+func (e *phonebookImpl) ReplacePeerList(dnsAddresses []string, networkName string) {
```

Comment:
I like **they** more than **dnsAddresses** - the latter assumes addresses come from DNS/SRV but they can also come from a config file

Reply:
changing to addressesThey